### PR TITLE
Modified to accept process.env

### PR DESCRIPTION
**Pull request by blackheat**

### CODE REVIEW

File: javascript/android-web-test.js [hunk 1]
```diff
@@ -3,8 +3,10 @@ import 'colors'
 import wd from 'wd'
 import {assert} from 'chai'
 
-const username = 'your Kobiton username'
-const apiKey = 'your Kobiton api key'
+const username = process.env.KOBITON_USERNAME
+const apiKey = process.env.KOBITON_API_KEY
+const deviceName = process.env.KOBITON_DEVICE_NAME || 'Galaxy*'
+const deviceOS = process.env.KOBITON_DEVICE_OS || 'Android'
```
Comment (NguyenHoanTien) [inline javascript/android-web-test.js]: const sessionName: process.env.KOBITON_SESSION_NAME || 'Sample session name'

File: javascript/android-web-test.js [hunk 2]
```diff
@@ -13,18 +15,23 @@ const kobitonServerConfig = {
...
-  sessionName:        'Automation test session',
-  sessionDescription: 'This is an example for Android web', 
-  deviceOrientation:  'portrait',  
-  captureScreenshots: true, 
-  browserName:        'chrome', 
-  deviceGroup:        'KOBITON', 
-  deviceName:         'Galaxy',
-  platformName:       'Android'
+  sessionName:        process.env.KOBITON_SESSION_NAME,
+  sessionDescription: process.env.KOBITON_SESSION_DESCRIPTION,
```
Comment (tiennguyenhoan) [inline javascript/android-web-test.js]: same

File: javascript/android-web-test.js [hunk 3]
```diff
@@ -13,18 +15,23 @@ const kobitonServerConfig = {
...
+  deviceOrientation:  process.env.KOBITON_SESSION_DEVICE_ORIENTATION,
```
Comment (tiennguyenhoan) [inline javascript/android-web-test.js]: same

File: javascript/android-web-test.js [hunk 4]
```diff
@@ -13,18 +15,23 @@ const kobitonServerConfig = {
...
+  captureScreenshots: process.env.KOBITON_SESSION_CAPTURE_SCREENSHOTS,
```
Comment (tiennguyenhoan) [inline javascript/android-web-test.js]: same

File: javascript/android-web-test.js [hunk 5]
```diff
@@ -13,18 +15,23 @@ const kobitonServerConfig = {
...
+  deviceGroup:        process.env.KOBITON_SESSION_DEVICE_GROUP,
```
Comment (tiennguyenhoan) [inline javascript/android-web-test.js]: same

File: javascript/android-web-test.js [hunk 6]
```diff
@@ -13,18 +15,23 @@ const kobitonServerConfig = {
...
+  deviceName:         deviceName,
+  platformName:       deviceOS,
+  browserName:        process.env.KOBITON_SESSION_BROWSER_NAME
```
Comment (tiennguyenhoan) [inline javascript/android-web-test.js]: same

**Review by tiennguyenhoan — CHANGES_REQUESTED** [review]
@blackheat please take a look

File: javascript/android-web-test.js [hunk 7]
```diff
@@ -13,18 +15,23 @@ const kobitonServerConfig = {
...
 const desiredCaps = {
-  sessionName:        'Automation test session',
-  sessionDescription: 'This is an example for Android web', 
-  deviceOrientation:  'portrait',  
-  captureScreenshots: true, 
-  browserName:        'chrome', 
-  deviceGroup:        'KOBITON', 
-  deviceName:         'Galaxy',
-  platformName:       'Android'
+  sessionName:        process.env.KOBITON_SESSION_NAME,
```
Comment (tiennguyenhoan) [inline javascript/android-web-test.js]: const name = process.env.KOBITON_SESSION_NAME  || ''

**Review by tiennguyenhoan — CHANGES_REQUESTED** [review]

**Comment by tiennguyenhoan** [comment]
Your PR sucks !!

**Pull request closed without merging**